_platforms/arm.md

```diff
@@ -0,0 +1,23 @@
+---
+name: arm
+date: 1970-01-01 04:00:00
+devices: [beaglebone, cubieboard2, odroid-c2, rpi, rpi2, rpi3, usbarmory]
+generic_devices: [armv6l, armv7l, aarch64]
+---
+
+{% capture download_details %}
+Live images can be written onto an SD card (i.e. using `dd`) and they allow you to have a ready to boot system. These images are prepared for 2GB SD cards. Alternatively, use the rootfs tarballs if you want to customize the partitions and filesystems.
+
+Connect to it in virtual terminal or via ssh and log in as `root`, password `voidlinux`.
+
+General and platform specific instructions for these images are available [in the documentation](https://docs.voidlinux.org/installation/guides/arm-devices/index.html).
+
+Deprecated instructions for the following platforms can be found in these wiki pages (you can help by porting them over to the [Void Handbook](https://github.com/void-linux/void-docs/blob/master/CONTRIBUTING.md)):
+
+- [BeagleBone/BeagleBone Black](https://wiki.voidlinux.org/Beaglebone)
+- [Cubieboard2](https://wiki.voidlinux.org/Cubieboard2_SD-Card)
+- [Odroid U2/U3](https://wiki.voidlinux.org/Odroid_U2)
+- [USB Armory](https://wiki.voidlinux.org/USB_Armory)
+{% endcapture %}
+
+{% include download_arm.html content=download_details %}
```

Comment:
I think we are losing some of the information here, such as the link to the page about RPis in the handbook. I also wouldn't like to lose the device information for each board (like `ARMv7, hard float` we currently have). Not sure if it could fit inside this, though.

Reply:
I have added arch back without `hard float` since all devices are `hard float`. I have removed a link to the RPI docs because the copy before it points to the related doc page (`General and platform specific instructions for these images are available in the documentation.`), I think this should be enough in this case. WDYT?

Reply:
Sounds great.